recipes/systemc/2.3.3/conanfile.py

```diff
@@ -0,0 +1,90 @@
+        "enable_pthreads": [True, False]
+    }
+    default_options = {
+        "shared": True,
+        "disable_async_updates": False,
+        "disable_copyright_msg": False,
+        "disable_virtual_bind":  False,
+        "enable_assertions": True,
+        "enable_immediate_self_notifications": False,
+        "enable_pthreads": False
+    }
+    generators = "cmake"
+    exports_sources = "sc_string_view.patch"
+
+    def configure(self):
+
+        if tools.valid_min_cppstd(self, "17"):
+            raise ConanInvalidConfiguration(
+                "C++ Standard %s not supported by SystemC" %
+                self.settings.compiler.cppstd)
+
+        if not self.settings.compiler.cppstd:
+            if tools.valid_min_cppstd(self, "14"):
+                self.settings.compiler.cppstd = 14
+            elif tools.valid_min_cppstd(self, "11"):
+                self.settings.compiler.cppstd = 11
+            elif tools.valid_min_cppstd(self, "98"):
+                self.settings.compiler.cppstd = 98
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        tools.replace_in_file("{}-{}/CMakeLists.txt".format(self.name,
+                                                            self.version),
+                              "project (SystemCLanguage CXX C)",
+                              """project (SystemCLanguage CXX C)
+                              include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+                              conan_basic_setup()""")
+        tools.patch(base_path="{}-{}".format(self.name, self.version),
+                    patch_file="sc_string_view.patch")
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.definitions["CMAKE_CXX_STANDARD"] = self.settings.compiler.cppstd
+        cmake.definitions["DISABLE_ASYNC_UPDATES"] = \
+            self.options.disable_async_updates
+        cmake.definitions["DISABLE_COPYRIGHT_MESSAGE"] = \
+            self.options.disable_copyright_msg
+        cmake.definitions["DISABLE_VIRTUAL_BIND"] = \
+            self.options.disable_virtual_bind
+        cmake.definitions["ENABLE_ASSERTIONS"] = \
+            self.options.enable_assertions
+        cmake.definitions["ENABLE_IMMEDIATE_SELF_NOTIFICATIONS"] = \
+            self.options.enable_immediate_self_notifications
+        cmake.definitions["ENABLE_PTHREADS"] = self.options.enable_pthreads
+        cmake.configure(source_folder="{}-{}".format(self.name, self.version))
+        cmake.build()
+        cmake.install()
+
+    def package(self):
+        tools.rmdir("{}/lib/cmake".format(self.package_folder))
+        tools.rmdir("{}/share".format(self.package_folder))
+        self.copy("LICENSE", dst="licenses",
+                  src="{}-{}".format(self.name, self.version))
+
+    def package_info(self):
+        self.cpp_info.libs = ["systemc"]
+
+        if (self.settings.os == "Linux") and not self.options.shared:
```

Comment:
Python doesn't need braces.
```suggestion
        if self.settings.os == "Linux" and not self.options.shared:
```